etc/hs-deps/stack-9.0.yaml

```diff
@@ -0,0 +1,9 @@
+resolver: lts-19.24
+packages:
+- '.'
+system-ghc: true
+install-ghc: true
+extra-deps:
+- megaparsec-9.3.0
```

Comment:
Can we remove any of these?

Reply:
Removed all except megaparsec - 9.3.0 adds better debugging so not essential, but nice to have

Reply:
perfect. just to be sure: this is a newer version of megaparsec than what's in the lts, right?

Reply:
As far as I can tell it's 9.2.2 in the LTS